### DISCUSSION TITLE
Won't let me update

### DISCUSSION THREAD

**Discussion by kennysden** [Security-Onion-Solutions/securityonion#2430] [Unsupported Versions] [answered]
I tried running sudo soup on my Security Onion and got this error:

Cloning Security Onion github repo into /tmp/sogh/securityonion.
Cloning into 'securityonion'...
Update was unable to pull from github. Please check your internet.

I tried pinging and accessing the web and I do have internet. Not sure what the issue is?

**Comment by TOoSmOotH** [comment, marked as answer]
I would try again. You can also check it by running curl https://github.com from the command line.

**Reply by kennysden** [reply]
I was able to curl github.com, but the sudo soup is still failing

**Reply by TOoSmOotH** [reply]
mkdir /tmp/test
cd /tmp/test
git clone https://github.com/Security-Onion-Solutions/securityonion.git
